### ISSUE TITLE
[BUG] Cannot use function tm_startswith

### ISSUE THREAD

**Issue by amkartashov** [terramate-io/terramate#1172]
**Describe the bug**

`terramate generate` fails with error saying `There is no function named "tm_startswith"`

Code:

```hcl
generate_hcl "gen.tf" {
  lets {
    test = tm_startswith(terramate.stack.path.absolute, "/")
  }
  content {
  }
}
```

**To Reproduce**

See <https://github.com/amkartashov/terramate_issue_1172.git>

```
git clone https://github.com/amkartashov/terramate_issue_1172.git && cd terramate_issue_1172
terramate generate
```

**Expected behavior**

No error is expected

**Log Output**

```
Code generation report

Failures:

- /test1
        error: /home/me/git/amkartashov/tm_bug/test1/gen.tm.hcl:3,12-25: lets eval: let.test: eval expression: There is no function named "tm_startswith".`
```

**Environment (please complete the following information):**
 - OS: Ubuntu 22.04 (WSL2)
 - Terramate Version 0.4.2

**Comment by i4ki** [comment]
Hi @amkartashov 

Thanks for reporting this. This was an oversight on my part when importing the Terraform documentation. This function is indeed not available in Terramate, as we use `v0.15.3` version of their library.

I will discuss with the team how to resolve that and reply ASAP.

**Comment by amkartashov** [comment]
w/a for those who seek solution: use `tm_substr`
